src/ripple/app/consensus/README.md

```diff
@@ -60,61 +55,61 @@ static std::uint32_t seqLeave;
 
 std::uint32_t getSequence() const;
 
-CxTime_t getCloseTime() const;
+Time_t getCloseTime() const;
 
-CxTime_t getSeenTime() const;
+Time_t getSeenTime() const;
 
-bool isStale (CxTime_t) const;
+bool isStale (Time_t) const;
 
-CxNodeID_t getNodeID() const;
+NodeID_t getNodeID() const;
 
-CxTxSetID_t getPosition() const;
+TxSetID_t getPosition() const;
 
-CxLgrID_t getPrevLedger() const;
+LgrID_t getPrevLedger() const;
 
 bool isInitial() const;
 
 bool isBowOut() const;
 
 Json::Value getJson() const;
 
-bool changePosition (CxTxSetID_t const& position, CxTime_t closeTime, CxTime_t now);
+bool changePosition (TxSetID_t const& position, Time_t closeTime, Time_t now);
 
-bool bowOut (CxTime_t now);
+bool bowOut (Time_t now);
 
 
-### `CxTx_t`
+### `Tx_t`
 
 Represents a transaction.
 Must provide:
 
-CxTxID_t getID() const;
+TxID_t getID() const;
 
 
-### CxTxSet_t
+### TxSet_t
 
 Represents a set of transactions.
 Must provide:
 
-CxTxSet_t (CxTxSet_t::mutable_t const&);
+TxSet_t (TxSet_t::mutable_t const&);
 
-CxTxSetID_t getID() const;
+TxSetID_t getID() const;
 
-bool hasEntry (CxTxID_t const&) const;
+bool hasEntry (TxID_t const&) const;
 
-bool hasEntry (CxTx_t const&) const;
+bool hasEntry (Tx_t const&) const;
 
-boost::optional<CxTx_t const> const getEntry (CxTxID_t const&) const;
+boost::optional<Tx_t const> const getEntry (TxID_t const&) const;
 
-CxTxSetDiff_t getDifferences(CxTxSet_t const&) const;
+TxSetDiff_t getDifferences(TxSet_t const&) const;
```

Comment:
Almost there. There is no `TxSetDiff_t`, so this needs to be. `std::map <TxID_t, bool>`